max7219/example_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2024 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package max7219_test
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"periph.io/x/conn/v3/driver/driverreg"
+	"periph.io/x/conn/v3/spi/spireg"
+	"periph.io/x/devices/v3/max7219"
+	"periph.io/x/host/v3"
+)
+
+// basic test program. To do a numeric display, set matrix to false and
+// matrixUnits to 1.
+func Example() {
+	var matrix = false
+	matrixUnits := 1
+	var err error
+	_, err = host.Init()
+	if err != nil {
+		log.Fatal(err)
+	}
+	if _, err = driverreg.Init(); err != nil {
+		log.Fatal(err)
+	}
```

Comment:
I'd recommend:

```
if _, err := host.Init(); err != nil {
  log.Fatal(err)
}
```

driverreg.Init() is redundant. err gets defined at line 32 so no need to define it explicitly.